tests/otel_integrations/test_celery.py

```diff
@@ -61,6 +61,7 @@ def celery_worker(celery_app: Celery) -> Iterator[WorkController]:
         yield worker
 
 
+@pytest.mark.xdist_group(name='sequential')
```

Comment:
I'm removing this one because I think this test is just flaky, I can't see evidence that xdist makes this fail more often.